src/appkit.cs

```diff
@@ -23721,4 +24387,592 @@ interface NSMenuValidation
+	}
+
+	[Mac (10,12,1)]
+	[BaseType (typeof(NSScrubberLayout))]
+	interface NSScrubberFlowLayout
+	{
+		[Export ("itemSpacing")]
+		nfloat ItemSpacing { get; set; }
+
+		[Export ("itemSize", ArgumentSemantic.Assign)]
+		CGSize ItemSize { get; set; }
+
+		[Export ("invalidateLayoutForItemsAtIndexes:")]
+		void InvalidateLayoutForItemsAtIndexes (NSIndexSet invalidItemIndexes);
+	}
+
+	[Mac (10,12,1)]
+	[BaseType (typeof(NSScrubberLayout))]
+	interface NSScrubberProportionalLayout
+	{
+		[Export ("numberOfVisibleItems")]
+		nint NumberOfVisibleItems { get; set; }
+
+		[Export ("initWithNumberOfVisibleItems:")]
+		[DesignatedInitializer]
+		IntPtr Constructor (nint numberOfVisibleItems);
+	}
+
+	public interface INSSharingServicePickerTouchBarItemDelegate {}
+
+	[Protocol]
+	interface NSSharingServicePickerTouchBarItemDelegate : NSSharingServicePickerDelegate
+	{
+		[Abstract]
+		[Export ("itemsForSharingServicePickerTouchBarItem:")]
+		INSPasteboardWriting [] ItemsForSharingServicePickerTouchBarItem (NSSharingServicePickerTouchBarItem pickerTouchBarItem);
+	}
+
+	[Mac (10,12,1)]
+	[BaseType (typeof(NSTouchBarItem))]
+	interface NSSharingServicePickerTouchBarItem
+	{
+		[Export ("initWithIdentifier:")]
+		[DesignatedInitializer]
+		IntPtr Constructor (string identifier);
+
+		[NullAllowed, Export ("delegate", ArgumentSemantic.Weak)]
+		INSSharingServicePickerTouchBarItemDelegate Delegate { get; set; }
+
+		[Export ("enabled")]
+		bool Enabled { [Bind ("isEnabled")] get; set; }
+
+		[Export ("buttonTitle")]
+		string ButtonTitle { get; set; }
+
+		[NullAllowed, Export ("buttonImage", ArgumentSemantic.Retain)]
+		NSImage ButtonImage { get; set; }
+	}
+
+	[Mac (10,12)]
+	[BaseType (typeof(NSObject))]
+	interface NSSliderAccessory : NSCoding, NSAccessibility, NSAccessibilityElement
+	{
+		[Static]
+		[Export ("accessoryWithImage:")]
+		NSSliderAccessory CreateAccessoryWithImage (NSImage image);
+
+		[Export ("behavior", ArgumentSemantic.Copy)]
+		NSSliderAccessoryBehavior Behavior { get; set; }
+
+		[Export ("enabled")]
+		bool Enabled { [Bind ("isEnabled")] get; set; }
+
+		[Mac (10, 12, 1)]
+		[Field ("NSSliderAccessoryWidthDefault")]
+		double WidthDefault { get; }
+
+		[Mac (10, 12, 1)]
+		[Field ("NSSliderAccessoryWidthWide")]
+		double WidthWide { get; }
+	}
+
+	[Mac (10,12)]
+	[BaseType (typeof(NSObject))]
+	interface NSSliderAccessoryBehavior : NSCoding, NSCopying
+	{
+		[Static]
+		[Export ("automaticBehavior", ArgumentSemantic.Copy)]
+		NSSliderAccessoryBehavior AutomaticBehavior { get; }
+
+		[Static]
+		[Export ("valueStepBehavior", ArgumentSemantic.Copy)]
+		NSSliderAccessoryBehavior ValueStepBehavior { get; }
+
+		[Static]
+		[Export ("valueResetBehavior", ArgumentSemantic.Copy)]
+		NSSliderAccessoryBehavior ValueResetBehavior { get; }
+
+		[Static]
+		[Export ("behaviorWithTarget:action:")]
+		NSSliderAccessoryBehavior CreateBehavior ([NullAllowed] NSObject target, Selector action);
```

Comment:
[Static]
		[Export ("behaviorWithTarget:action:")]
		NSSliderAccessoryBehavior CreateBehavior ([NullAllowed] NSObject target, Selector action);

		[Static]
		[Export ("behaviorWithHandler:")]
		NSSliderAccessoryBehavior CreateBehavior (Action<NSSliderAccessory> handler);


Wouldn't the 2nd one do that already @spouliot ?